pkg/resource/deploy/step_generator.go

```diff
@@ -783,5 +799,6 @@ func newStepGenerator(plan *Plan, opts Options) *stepGenerator {
 		deletes:              make(map[resource.URN]bool),
 		pendingDeletes:       make(map[*resource.State]bool),
 		dependentReplaceKeys: make(map[resource.URN][]resource.PropertyKey),
+		aliased:              make(map[resource.URN]resource.URN),
```

Comment:
can you document the direction here?  i.e. is this a mapping from the alias to the current URN, the reverse, something else?  etc.

Reply:
Done.